librz/include/rz_util/rz_table.h

```diff
@@ -55,6 +55,18 @@ typedef struct {
 
 typedef void (*RzTableSelector)(RzTableRow *acc, RzTableRow *new_row, int nth);
 
+struct rz_cons_printable_palette_t;
+
+/**
+ * \brief Visual rendering options for \ref rz_table_visual_list
+ */
+typedef struct rz_table_visual_options_t {
+	bool unicode; ///< Use Unicode runes for lines/blocks when true
```

Comment:
```suggestion
	bool unicode; ///< Use Unicode characters for lines/blocks when true
```

Runes is a term from go. Not defined in Unicode standard IIRC.